core/src/components/radio/radio.tsx

```diff
@@ -261,21 +270,12 @@ export class Radio implements ComponentInterface {
           'ion-activatable': !inItem,
           'ion-focusable': !inItem,
         })}
+        role="radio"
+        aria-checked={checked ? 'true' : 'false'}
+        aria-disabled={disabled ? 'true' : null}
+        tabindex={buttonTabindex}
       >
         <label class="radio-wrapper">
-          {/*
-            The native control must be rendered
-            before the visible label text due to https://bugs.webkit.org/show_bug.cgi?id=251951
-          */}
-          <input
```

Comment:
The native input was originally there to have an actual radio associated with the label and for focus management (i.e. tabbing to it with a keyboard).

However, since the host itself is the radio now this is no longer needed. Keeping it around would cause a nested interactive (since the host is a radio).

This does not cause issues with forms because the input that is submitted with forms is rendered in `ion-radio-group`, not `ion-radio`.

Reply:
Sounds good!